src/Polysemy/Writer.hs

```diff
@@ -27,10 +28,15 @@ import Polysemy.State
 data Writer o m a where
   Tell   :: o -> Writer o m ()
   Listen :: ∀ o m a. m a -> Writer o m (o, a)
-  Censor :: (o -> o) -> m a -> Writer o m a
+  Pass   :: m (o -> o, a) -> Writer o m a
```

Comment:
I flipped the elements in the tuple compared to MTL, since polysemy does so for `listen`, `runWriter`, etc. I have to assume that's reasonable, since nobody's going to use `pass` anyway.

Reply:
:+1: the original order in mtl is a bug, imo.